docs/markdown/Using Pants/concepts/targets.md

```diff
@@ -217,6 +217,19 @@ To reset any modified defaults, simply override with the empty dict:
     __defaults__(all={})
 ```
 
+Supporting optional plugin fields
+---------------------------------
+
+Normally Pants presents an error message when attempting to provide a default value for a field that doesn't exist for the target. However, some fields comes from plugins, and to support disabling a plugin without having to remove any default values referencing any plugin fields it was providing, there is a `ignore_unknown_fields` option to use:
+
+```python example/BUILD
+    __defaults__(
+      {
+        # Defaults...
+      },
+      ignore_unknown_fields=True,
+    )
+```
```

Comment:
I was contemplating whether this was a too niche option to get its own section like this.. wdyt?

Reply:
Yea, this seems fairly niche. If it were to be exposed to users, it seems like it should be via an option (similar to `--no-verify-config`)?

Reply:
I was thinking it may make sense to be able to provide it on a per declaration basis in order to not ignore more than absolutely necessary.